flatfs.go

```diff
@@ -864,16 +864,17 @@ func (fs *Datastore) checkpointLoop() {
 func (fs *Datastore) writeDiskUsageFile(du int64, doSync bool) {
 	tmp, err := ioutil.TempFile(fs.path, "du-")
 	removed := false
+	if err != nil {
+		log.Warningf("cound not write disk usage: %v", err)
+		return
+	}
+
```

Comment:
Nit: This should come even earlier, before imminently after the call, and before `removed := false`